### ISSUE TITLE
Billing optimization: allow fields parameter for Places API

### ISSUE THREAD

**Issue by kaYcee** [ubilabs/react-geosuggest#404]
### Short description
After sessionToken support (thanks!) implemented I spotted another source of undesired billing. I was surprised to be charged for every single Places API field (incl. atmospheric data) when using react-geosuggest.

### Expected results
I expected only certain fields to be delivered (well, just `formatted_address` in most cases).

### Actual results
All fields are delivered as specified by Google here: https://developers.google.com/maps/documentation/javascript/places

### Suggested solution
src/Geosuggest.tsx, lines 274+275 (`AutocompletionRequest`), 490-493 (`placesService.getDetails`).
```     
const options = {
        placeId: suggestToGeocode.placeId,
        sessionToken: this.sessionToken
--->    fields: .... // to be added and provided through props};
```

### Additional Information
Google provides the following usage warning:
`Warning: If you do not specify at least one field with a request, or if you omit the fields parameter from a request, ALL possible fields will be returned, and you will be billed accordingly.`

**Comment by ro-ka** [comment]
Thanks for bringing this up! Feel free to implement this.  😃

**Comment by joshgeller** [comment]
Would love to see this feature. @kaYcee, do you have bandwidth to submit a PR?

**Comment by lopezdp** [comment]
Man I have a feeling this is what I need to get my zip code...

**Comment by ro-ka** [comment]
This has been solved by #414. Release will be out in a moment.  😃
